tests/cover/test_integer_ranges.py

```diff
@@ -47,7 +47,7 @@ def do_draw(self, data):
 def test_intervals_shrink_to_center(inter, rnd):
     lower, center, upper = inter
     s = interval(lower, upper, center)
-    with settings(database=None, max_shrinks=2000):
+    with settings(database=None, max_shrinks=2000, timeout=-1):
```

Comment:
Tangential to the patch at hand, but I’m not a fan of negative timeout = no timeout. Why not use `if timeout is None: timeout = 0`?